test/__init__.py

```diff
@@ -0,0 +1,8 @@
+#
+# Copyright (C) 2025-present ScyllaDB
+#
+# SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
+#
+import os
+
+TEST_RUNNER = os.environ.get("SCYLLA_TEST_RUNNER", "pytest")
```

Comment:
I complained in a different PR that tried to introduce arbitrary variables  `__init__.py` and here I see it again (you guys will have a conflict in your PRs though). But I am giving up this battle :-( Looks like we'll have code in `__init__.py` now. Oh, well.